content/v2.0/visualize-data/other-tools/grafana.md

```diff
@@ -11,62 +10,48 @@ weight: 201
 v2.0/tags: [grafana]
 ---
 
-Use the [Grafana Flux Data Source plugin](https://grafana.com/grafana/plugins/grafana-influxdb-flux-datasource)
-to connect Grafana to InfluxDB 2.0 or InfluxDB Cloud 2.0.
+Use [Grafana](https://grafana.com/) to visualize data from **InfluxDB 2.0** and **InfluxDB Cloud**.
 
-{{% warn %}}
-The **Grafana Flux Data Source plugin** is currently in beta.
-{{% /warn %}}
+{{% note %}}
+The instructions in this guide require **Grafana Cloud** or **Grafana v7.1+**.
+{{% /note %}}
 
 1. [Sign up for {{< cloud-name >}}](/v2.0/get-started/) or
-   [Start InfluxDB 2.0 OSS](/v2.0/get-started/#start-with-influxdb-oss).
-2. [Download and install Grafana](https://grafana.com/grafana/download).
-3. Install the **Grafana Flux Data Source plugin** manually or with `grafana-cli` (installed
-   with Grafana).
-
-    {{< tabs-wrapper >}}
-    {{% tabs %}}
-[Install with Grafana CLI](#)
-[Install manually](#)
-    {{% /tabs %}}
-    {{% tab-content %}}
-```sh
-grafana-cli plugins install grafana-influxdb-flux-datasource
-```
-    {{% /tab-content %}}
-    {{% tab-content %}}
-  Download the [latest release](https://github.com/grafana/influxdb-flux-datasource/releases)
-  of the Flux Datasource Plugin into the Grafana `data/plugins` directory and unzip it.
-    {{% /tab-content %}}
-    {{< /tabs-wrapper >}}
-
-4. [Start Grafana](https://grafana.com/docs/grafana/latest/installation/) and
-   visit `http://localhost:3000` in your browser.
-5. In the left navigation of the Grafana user interface (UI), hover over the gear
+   [start InfluxDB 2.0 OSS](/v2.0/get-started/#start-with-influxdb-oss).
+2. [Sign up for Grafana Cloud](https://grafana.com/products/cloud/) or
+   [download and install Grafana](https://grafana.com/grafana/download).
+3. Visit your **Grafana Cloud user interface** (UI) or, if running Grafana locally,
+   [start Grafana](https://grafana.com/docs/grafana/latest/installation/) and visit
+   `http://localhost:3000` in your browser.
+5. In the left navigation of the Grafana UI, hover over the gear
    icon to expand the **Configuration** section. Click **Data Sources**.
 6. Click **Add data source**.
-7. Select **Flux (InfluxDB) [BETA]** from the list of available plugins.
+7. Select **InfluxDB** from the list of available data sources.
 
 ## Configure your InfluxDB 2.0 connection
-1. Enter a name for your Flux datasource.
-2. Under **HTTP**, enter your [InfluxDB URL](/v2.0/reference/urls/).
-3. Under **Auth**, select **With Credentials**.
-4. Under **InfluxDB 2.0.0 Details**, enter your **organization name** _or_ **organization ID**,
-   **default bucket**, and **authentication token**.
+1. Enter a **name** for your InfluxDB data source.
```

Comment:
How do I start to configure a connection? Where do I enter the InfluxDB data source name?